src/Reader/Http/LaminasHttpClientDecorator.php

```diff
@@ -47,10 +46,12 @@ public function get($uri, array $headers = [])
         }
         $response = $this->client->send();
 
+        $headers = $response->getHeaders()->toArray() ?? [];
```

Comment:
Im theory this is always an array?

Reply:
Eh yes, forgot the `?->`, will fix it.

Reply:
Oh I see what you mean now. `AbstractMessage` (Laminas\Http) handles null. IDE was showing it `null|Headers` for whatever reason. `null` is handled on getHeaders.